mmengine/dataset/dataset_wrapper.py

```diff
@@ -28,12 +28,15 @@ class ConcatDataset(_ConcatDataset):
     Args:
         datasets (Sequence[BaseDataset] or Sequence[dict]): A list of datasets
             which will be concatenated.
+        ignore_keys (List[str] or str): Ignore the keys that can be
+            unequal in `dataset.metainfo`.
         lazy_init (bool, optional): Whether to load annotation during
             instantiation. Defaults to False.
     """
 
     def __init__(self,
                  datasets: Sequence[Union[BaseDataset, dict]],
+                 ignore_keys: Union[str, List[str]] = [],
```

Comment:
Suggest not using a mutable variable as the default value, unless it is necessary. Maybe we could make it optional, if `ignore_key` is None, then convert it to an empty list.